e2e/runner/bitcoin.go

```diff
@@ -286,6 +296,57 @@ func (r *E2ERunner) SendToTSSFromDeployerWithMemo(
 	return txid, nil
 }
 
+func (r *E2ERunner) InscribeToTSSFromDeployerWithMemo(
+	amount float64,
+	inputUTXOs []btcjson.ListUnspentResult,
+	memo []byte,
+) (*chainhash.Hash, error) {
+	builder := InscriptionBuilder{sidecarUrl: "http://bitcoin-node-sidecar:8000", client: http.Client{}}
```

Comment:
Can we use a constant for this URL?

We generally also put urls in the config but in this case I think it is fine as it will be temporary